petclinic-frontend/src/features/inventories/InventoriesListTable.tsx

```diff
@@ -547,15 +549,50 @@ export default function InventoriesListTable(): JSX.Element {
           </div>
         </>
       )}
-      <button className="btn btn-primary" onClick={toggleAddSupplyModal}>
-        Add Supply
+      <button className="btn btn-primary" onClick={toggleAddProductModal}>
+        Add Product
       </button>
-      {showAddSupplyModal && (
-        <AddSupplyForm
-          onClose={toggleAddSupplyModal}
-          onSubmit={handleAddSupplySubmit}
+      {showAddProductModal && (
+        <AddProductForm
+          onClose={toggleAddProductModal}
+          onSubmit={handleAddProductSubmit}
         />
       )}
+      <div className="card-container-custom">
+        {inventoryList.map(inventory => (
+          <div
+            className="card"
+            key={inventory.inventoryName}
+            onClick={() =>
+              navigate(`/inventory/${inventory.inventoryId}/products`)
+            }
+            style={{ cursor: 'pointer' }}
+          >
+            <div className="image-container">
+              <img
+                src={inventory.inventoryImage}
+                alt={inventory.inventoryName}
+                className="card-image"
+                onError={(e: React.SyntheticEvent<HTMLImageElement, Event>) => {
+                  const target = e.target as HTMLImageElement;
+                  if (inventory.inventoryBackupImage) {
+                    target.src = inventory.inventoryBackupImage;
+                    target.onerror = () => {
+                      target.onerror = null;
+                      target.src = DefaultInventoryImage;
+                    };
+                  } else {
+                    target.src = DefaultInventoryImage;
+                  }
+                }}
+              />
+            </div>
+            <h2>{inventory.inventoryName}</h2>
+            <p>Type: {inventory.inventoryType}</p>
+            <p>{inventory.inventoryDescription}</p>
+          </div>
+        ))}
+      </div>
```

Comment:
I think you had the right idea when it came to have a desired image and a backup image to be shown in the frontend. That way, the user will always have a view to fall back on.